openedx/custom/db/fields.py

```diff
@@ -0,0 +1,79 @@
+from django.core import exceptions
+from django.utils.text import capfirst
+from multiselectfield import MultiSelectField
+from multiselectfield.db.fields import MSFList
+from multiselectfield.utils import string_type, get_max_length
+
+from openedx.custom.forms.fields import MultiSelectWithOtherFormField
+
+
+class MultiSelectWithOtherField(MultiSelectField):
+
+    def __init__(self, other_max_length=None, *args, **kwargs):
+        self.other_max_length = other_max_length
+
+        if kwargs.get('max_length') is None and self.other_max_length is not None:
```

Comment:
self. can be avoided, as argument `other_max_length` will have same value. So let's please remove self.